cmd/analyzer/parse_args.go

```diff
@@ -60,23 +60,40 @@ const (
 	allSubnetsDiff   = "diff_all_subnets"   // semantic diff of allSubnets analysis between two configurations
 )
 
-var supportedOutputFormats = map[string]bool{
-	JSONFormat:       true,
-	TEXTFormat:       true,
-	MDFormat:         true,
-	DRAWIOFormat:     true,
-	ARCHDRAWIOFormat: true,
-	DEBUGFormat:      true,
+var supportedOutputFormats = map[string][]string{
+	JSONFormat:       {allEndpoints, allSubnets},
+	TEXTFormat:       {allEndpoints, allSubnets, singleSubnet, allEndpointsDiff, allSubnetsDiff},
+	MDFormat:         {allEndpoints, allEndpointsDiff, allSubnetsDiff},
+	DRAWIOFormat:     {allEndpoints},
+	ARCHDRAWIOFormat: {allEndpoints},
+	DEBUGFormat:      {allEndpoints},
 }
-var supportedAnalysisTypes = map[string]bool{
-	allEndpoints:     true,
-	allSubnets:       true,
-	singleSubnet:     true,
-	allEndpointsDiff: true,
-	allSubnetsDiff:   true,
+var supportedAnalysisTypes = map[string][]string{
+	allEndpoints:     {TEXTFormat, JSONFormat, MDFormat, DRAWIOFormat, ARCHDRAWIOFormat, DEBUGFormat},
+	allSubnets:       {TEXTFormat, JSONFormat},
+	singleSubnet:     {TEXTFormat},
+	allEndpointsDiff: {TEXTFormat, MDFormat},
+	allSubnetsDiff:   {TEXTFormat, MDFormat},
 }
 
-func getSupportedValuesString(supportedValues map[string]bool) string {
+func getSupportedAnalysisTypesString() string {
+	valuesList := make([]string, len(supportedAnalysisTypes)+1)
+	i := 0
+	for key := range supportedAnalysisTypes {
+		j := 0
+		valuesString := make([]string, len(supportedAnalysisTypes[key]))
+		for _, value := range supportedAnalysisTypes[key] {
+			valuesString[j] = value
+			j += 1
```

Comment:
why do you need to copy into `valuesString` the list values from `supportedAnalysisTypes[key]`?
can just use `supportedAnalysisTypes[key]` instead of defining a new list and copying